readthedocs/doc_builder/python_environments.py

```diff
@@ -323,7 +323,7 @@ def setup_base(self):
             # Don't use virtualenv bin that doesn't exist yet
             bin_path=None,
             # Don't use the project's root, some config files can interfere
-            cwd='$HOME',
+            cwd=None,
```

Comment:
What happens when `cwd=` is `None`? Wouldn't be better to default to `/tmp` or similar?

Reply:
It defaults to `RTD_DOCKER_WORKDIR` ($HOME replacement), it can be omitted now, but didn't want to lose the comment